### PR TITLE
fix: add AliMora83

### LINKED ISSUE #1826: [PROFILE ADDITION] <AliMora83>
### Profile screenshot

AliMora83

### PR DESCRIPTION
**Pull request by AliMora83**
resolves #1826

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/bylickilabs.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "image": "https://user-images.githubusercontent.com/109308073/208296882-ea239594-81ea-419f-ae6f-e3470ccc9a3e.png",
-  "issueId": 1573,
+  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
```
Comment (Panquesito7) [inline profiles/bylickilabs.json:2]: Why was this changed? 🤔

Comment (AliMora83) [reply profiles/bylickilabs.json:2]: I was adding my info, then I saw this file has been modified (not by me). So when I wanted to commit mine, that one was also on the commit page. So I tried to delete it🤣 It did not work, so I committed all and knew I had to explain🤩.

My apologies. 🙏🏼 Only the changes to my file should be accepted. Please restore what happened on that file🙏🏼

Comment (Panquesito7) [reply profiles/bylickilabs.json:2]: Do you need any help reverting the changes?
